tests/functional/report_viewer_api/test_run_filter.py

```diff
@@ -48,18 +48,23 @@ def test_filter_run_names(self):
 
         # Filter runs which name starts with `test_files_`.
         test_runs = self.__get_runs('test_files_')
-        self.assertEqual(len(test_runs), 2,
-                         "There should be two runs for this test.")
+        self.assertEqual(len(test_runs), 1,
+                         "There should be one run for this test.")
 
         # Run name filter is case insensitive.
         test_runs = self.__get_runs('Test_Files_')
-        self.assertEqual(len(test_runs), 2,
-                         "There should be two runs for this test.")
+        self.assertEqual(len(test_runs), 1,
+                         "There should be one run for this test.")
 
-        # Filter runs which name contains `files`.
+        # Filter runs which name contains `files_`.
         test_runs = self.__get_runs('files_')
-        self.assertEqual(len(test_runs), 2,
-                         "There should be two runs for this test.")
+        self.assertEqual(len(test_runs), 1,
+                         "There should be one run for this test.")
+
+        # Filter runs which name contains `test_files*`.
+        test_runs = self.__get_runs('test_files*')
```

Comment:
I think there could be more tests, e.g. for `**, %`.

Reply:
Okay, I've created a few more test cases.